sbhc.py

```diff
@@ -257,7 +258,14 @@ def gather_categories(self, categories):
             self.category_dict[key] = category_items + children
 
     def emit_swift(self):
-        translation_unit = TranslationUnit.from_source(self.file_path, args=["-ObjC"])
+        cmakeArgs = ["-ObjC"]
+        if platform.system() == "Darwin":                                        # is Mac
```

Comment:
Since this is a Mac-only project, we don't need the platform check.

Reply:
ACK

Reply:
@tingraldi Fixed the issue. I left the version check in the code, because who knows, there may be a poor developer on this planet anywhere who'd stumble upon this tool and still have to struggle with the old OS.